### ISSUE TITLE
Enable admin to disable email notification of certain type

### ISSUE THREAD

**Issue by mfcndw** [tootsuite/mastodon#15495]
### Pitch

Enable admin to disable email notification of certain type in site settings.
For example, disable user from turning email notification on for liked toot, or follow request.

### Motivation

My instance use a free tier email service. Usually it's not an issue, most of the people doesn't turn on email notification for everything, especially most frequently notified items, to avoid spamming themselves.

However today I got usage alert from my email service provider. Turns out it's because a user with a lot of follower recently migrated to my instance and locked their account and turned email notification on. Thus a ton of their following request flooded my email service. 

Now I can either let my new user doesn't receive account verification email, or pay a premium to the email service provider for quota I don't use for 99% of the time.

I PMed the user to ask them to turn off the email notif, and am thinking of disable this option permanently on frontend for my site. But this may come handy in future for smaller instance who's relying on free tier email service and doesn't want to be flooded by one or two user who just love emails.

**Comment by jakesullyneytiri** [comment]
This option should be available. since i'm using free trial at mailgun for 3 months so it includes 5,000 free mails. With notification on it will clearly use up the 5,000 mails in no time.

So yeah i hope this will be added so admin can turn of notification by email in admin panel so no users can enable it.

**Comment by Evehic** [comment]
Is there any update on this?
It seems users have some emails turned on by default and I'd like the default to be none for the quota reason
If it cant be done from Admin page, Im wondering if theres a tool which can access the datatbase directly and turn these notifications off?

**Comment by fofwisdom** [comment]
This feature is desperately needed. We have a lot of one-time users who come to our events and leave, and when they get an email notification they don't connect and report it as spam.

This blocks email delivery. Users who need email notifications don't get them or can't sign up for Mastodon.

**Comment by Mubelotix** [comment]
Mastodon just spammed hundreds of emails at once when I enabled SMTP. They were accumulating in the queue for weeks. Definitely not the expected behavior, I didn't want them to go off

**Comment by anampses** [comment]
As my instance grows larger and larger in active userbase, I have observed a significant increase in cost with my email service provider. I am growing concerned about the scalability of this not keeping pace with the donation support I receive to keep the instance going and I would really like a way to disable specific email notifications from the admin panel.

**Comment by UvixCreative** [comment]
Adding that I'd also love this feature. I have a small community, but I only get 100 emails per day. I was only expecting to use emails for account registration, but now this could be 2-5x the usage I was expecting and could easily put me over my limit

**Comment by sgarciac** [comment]
It is quite hard nowdays to host your own MTA so most of us are stuck paying for a transactional mail service. Not being able to globally limit the set of email notifications a user can subscribe to is a big problem for small instances with small budgets.

**Comment by computer-8256** [comment]
Having a way to disable a part of the notification system would be really beneficial to us. at least, being able to change the default values of the preferences can make the situation easier.

**Comment by nemobis** [comment]
I agree with this proposal. If you're not trying to keep users addicted to the service, spending money on email notifications for activity rarely makes sense. 

ersonally I've set Mastodon to use send email through a local postfix instance. This allows some workarounds, however imperfect. You can configure postfix to [not forward email at all](https://serverfault.com/a/271443/203035), drop the mail queue in particular occasions (like the spammy events mentioned above), perhaps even silently drop email notifications of specific kinds as suggested above.

**Comment by TheUncannyObserver** [comment]
I would like to add my desire for this feature as well.

**Comment by trendless** [comment]
+1. Just had this happen today, almost five years after this issue was raised. There should at the very least be instance-wide defaults than can be set, so the few folks who need it can opt in, rather than admins having to ask everyone to opt out. Significant QoL improvement.

**Comment by smiba** [comment]
Would love to have this feature as well, I think allowing a user to set email notifications for every single interaction can be very expensive for operators. Users don't really realise this, I'd rather have it disabled entirely than having to play email cop

It's also an attack vector you could use to get a instance into trouble (either by reaching limits of mail plans, or financially)
